react.js

```diff
@@ -1,28 +1,48 @@
 module.exports = {
   env: {
-    browser: true
+    browser: true,
   },
-  parser: 'babel-eslint',
+  parser: "babel-eslint",
   parserOptions: {
     ecmaFeatures: {
-      jsx: true
-    }
+      jsx: true,
+    },
   },
-  plugins: ['react'],
+  plugins: ["react"],
   settings: {
     react: {
-      version: 'detect'
-    }
+      version: "detect",
+    },
   },
-  extends: [
-    'edlio/common',
-    'plugin:react/recommended'
-  ],
+  extends: ["edlio/common", "plugin:react/recommended"],
   rules: {
-    'react/display-name': 'off',
-    'react/no-unused-prop-types': 'error',
-    'react/no-unused-state': 'error',
-    'react/jsx-first-prop-new-line': ['error', 'multiline'],
-    'react/sort-comp': 'error'
-  }
-}
+    "react/display-name": "off",
+    "react/no-unused-prop-types": "error",
+    "react/no-unused-state": "error",
+    "react/jsx-first-prop-new-line": ["error", "multiline"],
+    "react/sort-comp": "error",
+    semi: "error",
```

Comment:
The same question https://github.com/edlio/cms/pull/14233#pullrequestreview-1762178787